### DISCUSSION TITLE
italian language

### DISCUSSION THREAD

**Discussion by Simone90** [PromtEngineer/localGPT#477] [Q&A]
Hi.
Which changes should I make to the repo in order to train on italian documents and answer in italian as well.
Thanks for the help.

**Comment by Scox-72** [comment]
Hi Simone,
Stefano Scotta published a model and a dataset for "RAI - Centre for Research, Technological Innovation and Experimentation" ([https://huggingface.co/raicrits]). 

I think you can use the model customizing "constants.py" with:
MODEL_ID = "raicrits/Hermes7b_ITA"
MODEL_BASENAME = None
